elements/pfe-progress-indicator/src/pfe-progress-indicator.scss

```diff
@@ -15,12 +15,14 @@ $pfe-progress-indicator--border-dark: #{pfe-var(surface--border-width--heavy)} #
     -webkit-animation:spin 1s linear infinite;
     -moz-animation:spin 1s linear infinite;
 
-    border-bottom: $pfe-progress-indicator--border-light;
-    border-left: $pfe-progress-indicator--border-light;
-    border-right: $pfe-progress-indicator--border-light;
-    border-top: $pfe-progress-indicator--border-dark;
+    border-style: $pfe-progress-indicator--style;
+    border-width: $pfe-progress-indicator--width;
+    // TBD: potentially make default color align with PatternFly, which is #06c
```

Comment:
Maybe we can label this with a `@todo: 2.0 ` decorator.  We have tooling that will pull out todos into a list.